subsys/bluetooth/services/hrs.c

```diff
@@ -91,6 +101,34 @@ static int hrs_init(void)
 	return 0;
 }
 
+int bt_hrs_cb_register(struct bt_hrs_cb *cb)
+{
+	if (cb == NULL) {
```

Comment:
Suggest to use `CHECKIF` for input validation

```suggestion
	CHECKIF(cb == NULL) {
```

Not necessary though, as we don't have any specific guidelines for this

Reply:
Updated